apps/drupal/webpack.config.js

```diff
@@ -6,25 +6,40 @@ const path = require('path');
 const { DefinePlugin } = require('webpack');
 
 // Plugins
-// const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const RunScriptAfterEmit = require('../../tools/webpack/run-script-after-emit');
 const particle = require('../../particle');
 
 // Constants: environment
 const { NODE_ENV } = process.env;
 // Constants: root
-const { PATH_DIST } = require('../../config');
+const { ASSETS_ATOMIC_FOLDER } = require('../../config');
 // Constants: app
-const { APP_NAME, APP_DESIGN_SYSTEM } = require('./config');
+const appConfig = require('./config');
```

Comment:
Should we have these filenames be more descriptive as to what config they are for? It makes sense being in the actual folder, but some of that I think gets lost when removed from the files and they are next to each other as in this case.

Reply:
Even if the file was named appConfig and we could immediately destructure of the resulting require module combining this line and line 19 below.

Reply:
I was thinking of moving these file names to something like `particle.app.config.js` to make it clear this is a particle-only file and this is specific to app config. The root file can then become `particle.config.js`.

Reply:
Looks great.